Pod/Classes/EigenCommunications/ARNotificationsManager.m

```diff
@@ -0,0 +1,26 @@
+#import "ARNotificationsManager.h"
+
+@implementation ARNotificationsManager
+
+RCT_EXPORT_MODULE();
+
+- (instancetype)init;
+{
+    if ((self = [super init])) {
+        [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(paymentRequestPaidReceived:) name:@"ARPaymentRequestPaid" object:nil];
```

Comment:
I’m thinking that maybe to decouple this code more from Eigen, we should not listen for `NSNotification`s here, but instead let Eigen in its Emission integration listen for its own notifications and then forward those to this event emitter through a normal method message send.

Reply:
Aye, this could be a promise - you can see the pattern inside the gene refine section